e2e/test/scenarios/admin/datamodel/datamodel.cy.spec.js

```diff
@@ -880,41 +880,56 @@ describe("scenarios > admin > datamodel > segments", () => {
       modal().contains("button", "Retire").click();
     });
 
-    it("should show segment revision history (metabase#45577)", () => {
+    it("should show segment revision history (metabase#45577, metabase#45594)", () => {
       cy.request("PUT", "/api/segment/1", {
         description: "Medium orders",
         revision_message: "Foo",
       });
 
+      cy.log("Make sure revisions are displayed properly in /references");
+      cy.visit("/reference/segments/1/revisions");
```

Comment:
Please avoid multiple `cy.visit` calls in a single test unless it's needed for repro - it's the most expensive action.
The test will almost always run faster when _clicking_ through UI instead.

Reply:
Got it, but in this case this was the most effective path.